apps/juxtaposition-ui/src/services/juxt-web/routes/admin/admin.js

```diff
@@ -153,6 +170,41 @@ router.post('/accounts/:pid', async (req, res) => {
 			link: '/titles/2551084080/new'
 		});
 	}
+
+	let action = 'UPDATE_USER';
+	const changes = [];
+	if (oldUserSettings.account_status !== req.body.account_status) {
+		switch (req.body.account_status) {
+			case 0:
+				action = 'UNBAN';
+				break;
+			case 1:
+				action = 'LIMIT_POSTING';
+				break;
+			case 2:
+				action = 'TEMP_BAN';
+				break;
+			case 3:
+				action = 'PERMA_BAN';
+				break;
+		}
+		changes.push(`Account_status changed from "${oldUserSettings.account_status}" to "${req.body.account_status}"`);
```

Comment:
Might be nice to render the `action` string here rather than the account status number. Also worth considering the mods sometimes experiment with invalid account statuses (-1, 4 and above) with inspect element, which will show up as `UPDATE_USER` right now despite being treated as a permanent ban by the code. So maybe just add `default` to the `PERMA_BAN` case